modules/SetTime.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/prefer-default-export *//* eslint-disable import/prefer-default-export */
+import { DateTime } from './luxon/time.js';
+
+const siteDate = document.querySelector('.time');
+export const currentTime = () => {
+  const theTime = DateTime.now();
+  siteDate.innerHTML = `${theTime.c.day}/${theTime.c.month}/${theTime.c.year}`;
+};
```

Comment:
- [ ] Kindly display the time and date and not only the date. You can format it by doing this ` const theTime = DateTime.now().toFormat("ff' UTC'Z")` or any other format you want to use but please display both the time and date. 👍